tutorials/cp-workflow-getting-started/cp-workflow-getting-started.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Prepare to create workflows in SAP Cloud Platform
+description: Enable and configure the services involved and assign the user roles you'll need for workflows.
+primary_tag: products>sap-cloud-platform
+tags: [  tutorial>beginner, topic>cloud, products>sap-cloud-platform ]
+---
+
+## Prerequisites  
+ - **Proficiency:** Beginner
+ - **Tutorials:** [Sign up for a free trial account on SAP Cloud Platform](https://www.sap.com/developer/tutorials/hcp-create-trial-account.html)
+
+## Details
+### You will learn  
+- To enable the workflow service in your SAP Cloud Platform account.
+- To assign the roles you need to develop and use workflow applications.
+
+### Time to Complete
+**5 Min**
+
+---
+
+[ACCORDION-BEGIN [Step 1: ](Open the SAP Cloud Platform Cockpit)]
+
+1. In your Web browser, open the [SAP Cloud Platform](https://account.hanatrial.ondemand.com/cockpit) cockpit. If you do not have a trial account, see Prerequisites.
+2. Choose **Neo Trial**.
+![Choose Neo Trial](choose-neo-trial.png)
+
+3. Select **Services** from the left-hand navigation.
+![Select Service](select-services.png)
+
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 2: ](Enable the Portal service for your account)]
+
+Search for the **Portal** service. Then select it, and choose **Enable**.
+![Enable the Portal service](portal-enabled.png)
+
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 3: ](Enable SAP Web IDE Full-Stack)]
+
+Search for **SAP Web IDE Full-Stack**. Then select it, and choose **Enable**.
+> Note: The workflow features are only available with SAP Web IDE Full-Stack.
+
+![Enable SAP Web IDE](webide-enabled.png)
+
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 4: ](Enable the workflow service)]
+
+Search for the **Workflow** service. Then select it, and choose **Enable**.
+![Enable the workflow service](workflow-enabled.png)
+
```

Comment:
And here I run in to a real problem.  The service failed to start.

Why?

![screen shot 2017-12-21 at 10 50 44 am](https://user-images.githubusercontent.com/3226389/34263281-e7ced706-e63c-11e7-8722-ce9b85d081c4.png)

Reply:
I added another note to the step: "> Note: If the enablement fails, try again – this could be a temporary problem."
I created an internal incident https://support.wdf.sap.corp/sap/support/message/1780524785